x-pack/solutions/security/plugins/security_solution/public/threat_intelligence/modules/indicators/components/common/field_value.tsx

```diff
@@ -30,9 +31,25 @@ export interface IndicatorFieldValueProps {
  * Takes an indicator object, a field and a field => type object to returns the correct value to display.
  * @returns If the type is a 'date', returns the {@link DateFormatter} component, else returns the value or {@link EMPTY_VALUE}.
  */
+
+const getMultipleFieldValues = (indicator: Indicator, fieldId: RawIndicatorFieldId) => {
+  if (!indicator) {
+    return null;
+  }
```

Comment:
Looking at the type for `indicator` I don't think it can be `null` so we should be able to remove this check